core/tools/generate_names.cc

```diff
@@ -95,6 +95,8 @@ NameDef names[] = {
     {"magic", "<magic>"},
     {"unconditional", "<unconditional>"},
     {"argPresent", "<argPresent>"},
+    {"rescueHeaderTemp", "<rescueHeaderTemp>"},
+    {"rescueTrueTemp", "<rescueTrueTemp>"},
```

Comment:
Can you add these to the magic function at https://github.com/sorbet/sorbet/blob/78116cac95a86afc13d5662e14db1ea2874bba92/core/LocalVariable.cc#L13-L28 ?

Or, possibly better yet, add special names that bracket these CFG temporaries:

```c++
    {"cfgTemporaryStart", "<cfgTemporaryStart>"},
    // existing temporaries go here
    {"cfgTemporaryEnd", "<cfgTemporaryEnd>"},

    ...
```

and make the above function do a range check on the ID of its argument to ensure it's between the IDs of those magic names?  (I think that would be best as a separate PR.)